src/tests/suite/daos_obj.c

```diff
@@ -1147,7 +1147,7 @@ iterate_records(struct ioreq *req)
 		if (number == 0)
 			continue;
 
-		for (i = 0; i < number; i++) {
+		for (i = 0; i < (number - 1); i++) {
```

Comment:
what is the purpose of the - 1 ?

Reply:
I can't seem to find the Coverity Issue that this would have stemmed from anymore.
I've pushed a change to remove it and will add comments to non-obvious changes like this in the future so I won't lose the original purpose.